env0/resource_template.go

```diff
@@ -120,6 +130,19 @@ func templateCreatePayloadFromParameters(d *schema.ResourceData) (client.Templat
 	if githubInstallationId, ok := d.GetOk("github_installation_id"); ok {
 		result.GithubInstallationId = githubInstallationId.(int)
 	}
+	if tokenId, ok := d.GetOk("token_id"); ok {
+		result.TokenId = tokenId.(string)
+	}
+	if gitlabProjectId, ok := d.GetOk("gitlab_project_id"); ok {
+		result.GitlabProjectId = gitlabProjectId.(int)
+	}
+
```

Comment:
does it work? i remember we do not save this on db

Reply:
5a4bc914

Reply:
I didnt thought you need to delete it, just tell me if it works or not - or if we SHOULD make it work?

Reply:
I don't think we should
I saw on our FE we don't get it so i guess its not returned from BE